plutus-tx-plugin/test/IntegerLiterals/NoStrict/NoNegativeLiterals/Spec.hs

```diff
@@ -1,17 +1,9 @@
-{-# LANGUAGE BangPatterns          #-}
-{-# LANGUAGE DataKinds             #-}
-{-# LANGUAGE FlexibleContexts      #-}
-{-# LANGUAGE MultiParamTypeClasses #-}
-{-# LANGUAGE NegativeLiterals      #-}
```

Comment:
This file had  both `NegativeLiterals` and `NoNegativeLiterals`, in that order.  Presumably the latter (which is what's wanted here) took precedence.